app/src/main/java/com/example/newsapp/ui/home/fragments/MoreFragment.kt

```diff
@@ -75,7 +75,7 @@ class MoreFragment : Fragment() {
             if (hasSaved) {
 
                 viewLifecycleOwner.lifecycleScope.launch {
-                    val listOfImages = loadImageFromStorage()
+                    val listOfImages = moreViewModel.loadImageFromStorage()
                     for (im in listOfImages) {
                         if (im.name.contains(moreViewModel.getUserEmail())) {
```

Comment:
@jnandrawos I think you can implement a better way for loading the user image here instead of passing through the list of saved files and checking the file name, why not just save the image path in the database along with the user object and use that value to load it directly ?

Reply:
done